src/platforms/common/usb_descriptors.h

```diff
@@ -65,7 +68,7 @@ static const usb_endpoint_descriptor_s gdb_comm_endp = {
 	.bEndpointAddress = (CDCACM_GDB_ENDPOINT + 1U) | USB_REQ_TYPE_IN,
 	.bmAttributes = USB_ENDPOINT_ATTR_INTERRUPT,
 	.wMaxPacketSize = 16,
-	.bInterval = 255,
+	.bInterval = MAX_BINTERVAL,
```

Comment:
This is technically fine, however we feel it might make a little more sense to name the new macro `USB_MAX_INTERVAL`, especially as the b prefix is not adding any information here

Reply:
Agreed, done!